Tests/ConfigurationManagerSpec.swift

```diff
@@ -90,6 +90,13 @@ class ConfigurationManagerSpec: QuickSpec {
                         configurationService.simulateRequestFailure = false
                     }
                 }
+
+                it("should retry for .tooManyRequestsError error") {
+                    configurationService.simulateRequestFailure = true
+                    configurationService.mockedError = .tooManyRequestsError
+                    configurationManager.fetchAndSaveConfigData(completion: { _ in })
+                    expect(configurationManager.scheduledTask).toEventuallyNot(beNil())
```

Comment:
Is there a way to kill this (and other) scheduled task after this evaluation? I didn;t think of it earlier

Reply:
Ok. don't worry about it for now. I'll take care of it later